xchange-gemini/src/main/java/org/knowm/xchange/gemini/v1/dto/trade/GeminiLimitOrder.java

```diff
@@ -10,6 +10,7 @@
  * Poloniex order response contains details of any trades that have just executed in the order entry return value. If a LimitOrder of this type is
  * supplied to the trade service orderEntry method it will be populated with this information.
  */
+@SuppressWarnings("serial")
```

Comment:
in general, I don't like these `suppressWarnings` in the code, as you can configure your IDE to not show warnings.